ZipBuilder/Sources/ZipBuilder/CarthageUtils.swift

```diff
@@ -22,6 +22,60 @@ import Foundation
 enum CarthageUtils {}
 
 extension CarthageUtils {
+  /// Package all required files for a Carthage release.
+  ///
+  /// - Parameters:
+  ///   - templateDir: The template project directory, contains the dummy Firebase library.
+  ///   - carthageJSONDir: Location of directory containing all JSON Carthage manifests.
+  ///   - artifacts: Release Artifacts from build.
+  ///   - rcNumber: The RC number.
+  /// - Returns: The path to the root of the Carthage installation.
+  static func packageCarthageRelease(templateDir: URL,
+                                     carthageJSONDir: URL,
+                                     artifacts: ZipBuilder.ReleaseArtifacts,
+                                     rcNumber: Int?) -> URL? {
+    if let zipLocation = artifacts.carthageDir {
+      var carthageRoot: URL?
```

Comment:
Nit: to avoid declaring `carthageRoot` as `var` you may wrap calculation of the value into a closure like here:
```
let carthageRoot: URL? = {
  do {
      return try doSomething()
  } catch {
      return nil
  }
}()
```